docs/concept/buckconfig.soy

```diff
@@ -2006,6 +2007,87 @@ cxx_library(
   {/param}
 {/call}
 
+{call buckconfig.section}
+  {param name: 'kotlin' /}
+  {param description}
+    This section configures various aspects of the Kotlin toolchain.
+  {/param}
+{/call}
+
+{call buckconfig.entry}
+  {param section: 'kotlin' /}
+  {param name: 'kotlinc' /}
+  {param example_value: '/usr/local/bin/kotlinc' /}
+  {param description}
+    The path to the <code>kotlinc</code> compiler executable to use when external compilation is
+    forced.  This setting has no effect by itself and must be paired with the <code>external</code>
```

Comment:
You should link to this (even if it's directly below it today, a link will always work even if code moves away).  We have a macro to construct links to config entries.